lib/reversetunnel/agent.go

```diff
@@ -659,10 +692,12 @@ func (a *agent) handleDiscovery(ch ssh.Channel, reqC <-chan *ssh.Request) {
 }
 
 const (
-	chanHeartbeat    = "teleport-heartbeat"
-	chanDiscovery    = "teleport-discovery"
-	chanDiscoveryReq = "discovery"
-	reconnectRequest = "reconnect@goteleport.com"
+	chanHeartbeat        = "teleport-heartbeat"
+	chanDiscovery        = "teleport-discovery"
+	chanDiscoveryReq     = "discovery"
+	chanNetworkConfig    = "teleport-netconfig"
+	chanNetworkConfigReq = "netconfig"
```

Comment:
This should be a global request rather than a channel, but either way:
```suggestion
	chanNetworkConfigReq = "netconfig@goteleport.com"
```